backend/layers/processing/requirements.txt

```diff
@@ -1,4 +1,4 @@
-cellxgene-schema
+git+https://github.com/chanzuckerberg/single-cell-curation/#subdirectory=cellxgene_schema_cli
```

Comment:
This one needs to be reverted at some point, but it might be a wise idea to keep this (at least) until we test on the whole corpus.